chia/full_node/full_node_api.py

```diff
@@ -1314,13 +1314,34 @@ async def request_puzzle_solution(self, request: wallet_protocol.RequestPuzzleSo
         return response_msg
 
     @api_request
-    async def request_header_blocks(self, request: wallet_protocol.RequestHeaderBlocks) -> Optional[Message]:
+    async def request_header_block_blobs(self, request: wallet_protocol.RequestHeaderBlockBlobs) -> Optional[Message]:
```

Comment:
We should add an extra param in the request called return_filter. The old protocol still uses the filter so it would be nice to still support returning it, even if we don't use it in current wallet

Reply:
but the old protocol would just use the old method no?

Reply:
Yeah but in the future we can depreciate the old one, but the filter might still be useful for other developers

Reply:
I'm not opposed to keeping both if it's a clean solution though